views/soa-tree-view/vite.config.ts

```
@@ -0,0 +1,23 @@
+import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js';
+
+export default defineConfig({
+  plugins: [
+    preact({
+      babel: {
+        plugins: [['@babel/plugin-proposal-decorators', { legacy: true }], ['@babel/plugin-proposal-class-properties']],
+      },
+    }),
+    ,
+    cssInjectedByJsPlugin(),
+  ],
```

Biome (2.4.4)

[error] This array contains an empty slots..

(lint/suspicious/noSparseArray)